packages/go_router/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: go_router
+description: A declarative router for Flutter based on Navigation 2 supporting
+  deep linking, data-driven routes and more
+version: 3.0.1
+repository: https://github.com/csells/go_router
```

Comment:
This may need to be update to 
```
repository: https://github.com/flutter/packages/tree/master/packages/go_router
issue_tracker: <flutter/flutter with labels>
```
See pubspec.yaml in other packages